src/hdmf/backends/io.py

```diff
@@ -46,6 +50,9 @@ def read(self, **kwargs):
             # TODO also check that the keys are appropriate. print a better error message
             raise UnsupportedOperation('Cannot build data. There are no values.')
         container = self.__manager.construct(f_builder)
+        if self.external_resources_path is not None:
```

Comment:
I think we should "safe-guard" reading the external resources with appropriate try/except blocks here so that a user can still get the container that is being read even if reading of the `ExternalResources` fails.

Reply:
raise warnings instead of errors as well?

Reply:
Yes, I think because this is in `HDMFIO.read` we'll need to raise warnings instead of errors here. The main reason is that we'd want the user to be able to read the Container even if reading the ExternalResources fails. Reading ExternalResources inside HDMFIO is in many ways a convenience and we would not want that functionality to break the core function of reading the main data file.

Reply:
I also saw your comment about moving ER before a release. Will do